templates/bonsai-starter/methods/src/main.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 RISC Zero, Inc.
```

Comment:
Having the `main.rs` be in the `methods` crate is a little bit of a deviation from our normal starter pattern of the `host` vs `methods` crates.